### ISSUE TITLE
Cannot call shared library steps with more than one parameter

### ISSUE THREAD

**Issue by RudolfVonKrugstein** [lesfurets/JenkinsPipelineUnit#44]
Hey,
It seems that I cannot call steps from a shared library, if they have more parameters. Example:

vars/onePar.groovy
```
def call(par) {
}
```

vars/twoPar.groovy
```
def call(par1,par2) {
}
```

When I now try to test this I can call "onePar". But with twoPar I get:

```
No signature of method: com.lesfurets.jenkins.unit.PipelineTestHelper$_setGlobalVars_closure9$_closure18$_closure19.call() is applicable for argument types: (java.lang.String, java.lang.String) values: [par1, par2, ]
Possible solutions: any(), any(), grep(), find(), dump(), collect()
	at TestScripts.testscript(TestScripts.groovy:121)
```

Is the number of parameters somehow hardcoded? What could this be?

Thanks!

**Comment by ozangunalp** [comment]
Which version of the framework are you using ? 

The bug #21 was already fixed in [release 1.0](https://github.com/lesfurets/JenkinsPipelineUnit/releases/tag/v1.0).

**Comment by RudolfVonKrugstein** [comment]
'For some reason I had version 0.12, my bad. Sorry for the noise.
